packages/client/src/lib/interface.ts

```diff
@@ -1,6 +1,8 @@
 import type { UnixFSEntry } from 'ipfs-car/unpack'
 import type { CID } from 'multiformats'
-export type { CID , UnixFSEntry }
+import type { Web3File } from 'web3-file'
+export type { CID , UnixFSEntry, Web3File }
```

Comment:
```suggestion
export type { CID, UnixFSEntry, Web3File }
```

Reply:
I will add linting to the project tomorrow to avoid these